include/internal/libspdm_common_lib.h

```diff
@@ -305,9 +304,23 @@ typedef struct {
     libspdm_chunk_info_t send;
     libspdm_chunk_info_t get;
 } libspdm_chunk_context_t;
-
 #endif /* LIBSPDM_ENABLE_CHUNK_CAP */
 
+#if LIBSPDM_ENABLE_MSG_LOG
+typedef struct {
+    void *buffer;
+    size_t buffer_size;
+    uint32_t mode;
+    size_t offset;
+    uint32_t status;
+} libspdm_msg_log_t;
```

Comment:
After read the code, I realize that buffer_size means max_buffer_size, while offset means real buffer_size. Suggest to change the name.